CHANGELOG.md

```
@@ -2,6 +2,143 @@
 
 All notable changes to GBrain will be documented in this file.
 
+## [0.27.0] - 2026-04-28
+
+## **GBrain runs on any embedding stack. OpenAI, Google Gemini, Ollama, Voyage, or anything via LiteLLM — one config line away.**
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)